AWS S3+STS & Azure Active Directory (Azure AD) OpenID Connect.cyberduckprofile

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2002-2023 iterate GmbH. All rights reserved.
+  ~ https://cyberduck.io/
+  ~
+  ~ This program is free software; you can redistribute it and/or modify
+  ~ it under the terms of the GNU General Public License as published by
+  ~ the Free Software Foundation, either version 3 of the License, or
+  ~ (at your option) any later version.
+  ~
+  ~ This program is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  -->
+
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>aws-s3-sts-azure-activedirectory-oidc</string>
+        <key>Description</key>
+        <string>AWS S3/STS &amp; Azure Active Directory (Azure AD)</string>
+        <key>Default Nickname</key>
+        <string>AWS S3/STS + Azure AD OIDC</string>
+        <key>Authorization</key>
+        <string>AuthorizationCode</string>
+        <key>OAuth Authorization Url</key>
+        <string>https://login.microsoftonline.com/common/oauth2/v2.0/authorize</string>
+        <key>OAuth Token Url</key>
+        <string>https://login.microsoftonline.com/common/oauth2/v2.0/token</string>
+        <key>OAuth Client ID</key>
+        <string>f40bc18f-cd02-4212-b7f1-15243e4e2ad3</string>
+        <key>OAuth Client Secret</key>
+        <string/>
+        <key>OAuth Redirect Url</key>
+        <string>${oauth.handler.scheme}://oauth</string>
+        <key>Scopes</key>
+        <array>
+            <string>openid</string>
+            <string>offline_access</string>
+        </array>
+        <key>Password Configurable</key>
+        <false/>
+        <key>Username Configurable</key>
+        <false/>
+        <key>Username Placeholder</key>
+        <string>Username</string>
+        <key>Properties</key>
+        <array>
+            <!-- Require Role ARN input from user -->
```

Comment:
When are users required to give this input? When creating the bookmark?

Reply:
The prompt is displayed when connecting and saved in the bookmark as a property. In https://github.com/iterate-ch/cyberduck/pull/14972/files#diff-df40981f50d48245937b3cc879c98c6185d507f12496bf65f1f96a5f18130db6R121-R132